### ISSUE TITLE
[BUG] Annotations are difficult to make use of

### ISSUE THREAD

**Issue by iainlane** [Sonia-corporation/stale#422]
**Describe the bug**

I just ran the action for the first time with `dry-run` to see what it would propose to do, and the annotations (~ the output) look like this

![Screenshot 2022-02-08 at 15 32 56](https://user-images.githubusercontent.com/321014/153020589-74e1d810-b5b4-40b9-921e-cb74cf0a52a6.png)

**Expected behaviour**

There are a couple of problems with this that I can see

  * They contain shell escapes
  * The workflow proposed to comment on 128 issues and 21 pull requests, but I can't see that many annotations there (perhaps 10 is a cap?)
  * Maybe it would be good if the annotations were a _summary_ of the actions to perform, and at the end of the full output the entire list was printed. I'd like to see what the action will do if I remove `dry-run`, but it's not that easy at the minute.

Thanks!

**Comment by C0ZEN** [comment]
@iainlane can you give me a public URL to check where do you see those annotations?
I never tested this, I searched inside this repository if I could see this UI, but unsuccessfully.

I am used to simply check out the logs [like here](https://github.com/Sonia-corporation/stale/runs/5108850276?check_suite_focus=true) to see if everything works well.

I can take a look tonight.

**Comment by iainlane** [comment]
@C0ZEN I'm sorry, I can't link to our run because it's a private repo 😭

But here's a run in this repo here which shows the bug: https://github.com/Sonia-corporation/stale/actions/runs/1717900568

Just from grepping the code I think it comes from `processor.logger.notice`, and some further chasing of documentation shows me it comes from [here](https://github.com/actions/toolkit/tree/main/packages/core#annotations). Anything you log via these will appear as annotations I think, shouldn't use colours, and should make sense as a "summary"! 🙂

**Comment by C0ZEN** [comment]
@iainlane yes, it's pretty clear to me what needs to be done.
I consider this a bug, since the documentation is not mentioning that annotations are incompatible with colours logs.
I will change how I use annotations to get rid of all sugar logs, and see how I can improve the `notice` to bring easy and useful insights.
Not only that, but I think being actually another way to just show some statistics is just a quick win.

**Comment by C0ZEN** [comment]
In the version that is currently in deployment, every single warning and error will no longer appear as an annotation but as a simple log, keeping the current colouring.
So, no difference will appear in the logs.

On top of this, I added brand-new warnings and errors annotations to easily display those important information, but without any kind of formatting.
Moreover, I removed every single notice annotation, but I plan to bring back new ones based on the statistics and outputs.
See #429
